### PR TITLE
Adds masthead utility links on the vertical nav on small viewports

### LINKED ISSUE #496: Vertical navigation hides any horizontal navigation items on narrow viewports
When the Vertical Navigation pattern is used with any menu items on the top navigation bar (status/alerts, help, user preferences/logout as shown at http://www.patternfly.org/pattern-library/navigation/vertical-navigation/vertical-navigation.html#_), and the viewport is narrow (such as on a smart phone, or Mozilla Firefox made narrow), the top navigation items are inaccessible.

### PR DESCRIPTION
**Pull request by andresgalante**
## Description
This PR is related to [this jira issue](https://patternfly.atlassian.net/browse/PTNFLY-1934) and fixes #496

You can preview it here:
https://rawgit.com/andresgalante/patternfly/vertical-nav-PTNFLY-1934-dist/dist/tests/vertical-navigation-primary-only.html

@jennyhaines @LHinson it does not reproduce the mockups for 2 reasons: 

1- Changing the colors to a darker black on the lower part of the vertical nav was extremely tricky when you start dealing with the nested navigations.
2- Since they are smaller, the hitting area is shorter than whats recommend (40px).

Can we play with our constrains for pf4 and implement it following the mockups for pf5?

@jeff-phillips-18 Please take a look at it, I don't really know how the submeus are generated and I fell like I have hardcoded it.

Thanks a lot ❤️

### CODE REVIEW

File: tests/pages/_includes/widgets/navigation/vertical-navigation.html [hunk 1]
```diff
@@ -57,7 +56,61 @@
...
+      </a>
+      <div id="user-secondary" class="nav-pf-secondary-nav">
+        <div class="nav-item-pf-header">
+          <a class="secondary-collapse-toggle-pf" data-toggle="collapse-secondary-nav"></a>
+          <span>User</span>
+        </div>
+
+        <ul class="list-group">
+          <li class="list-group-item " data-target="#amet-detracto-tertiary">
+            <a>
```
Comment (jeff-phillips-18) [inline tests/pages/_includes/widgets/navigation/vertical-navigation.html]: data-target here is not necessary.

File: tests/pages/_includes/widgets/navigation/vertical-navigation.html [hunk 2]
```diff
@@ -57,7 +56,61 @@
...
+              <span class="list-group-item-value">Preferences</span>
+            </a>
+          </li>
+
+          <li class="list-group-item " data-target="#amet-mediocrem-tertiary">
```
Comment (jeff-phillips-18) [inline tests/pages/_includes/widgets/navigation/vertical-navigation.html]: again, data-target is unnecessary

File: tests/pages/_includes/widgets/navigation/vertical-navigation.html [hunk 3]
```diff
@@ -57,7 +56,61 @@
...
+        <span class="pficon pficon-help" data-toggle="tooltip" title="" data-original-title="Amet"></span>
+        <span class="list-group-item-value">Help</span>
+      </a>
+      <div id="help-secondary" class="nav-pf-secondary-nav">
+        <div class="nav-item-pf-header">
+          <a class="secondary-collapse-toggle-pf" data-toggle="collapse-secondary-nav"></a>
+          <span>Help</span>
+        </div>
+        <ul class="list-group">
+          <li class="list-group-item " data-target="#amet-detracto-tertiary">
```
Comment (jeff-phillips-18) [inline tests/pages/_includes/widgets/navigation/vertical-navigation.html]: data-target...

File: tests/pages/_includes/widgets/navigation/vertical-navigation.html [hunk 4]
```diff
@@ -57,7 +56,61 @@
...
+            <a>
+              <span class="list-group-item-value">Help</span>
+            </a>
+          </li>
+          <li class="list-group-item " data-target="#amet-mediocrem-tertiary">
```
Comment (jeff-phillips-18) [inline tests/pages/_includes/widgets/navigation/vertical-navigation.html]: data-target...

**Comment by LHinson** [comment]
@andresgalante I'm ok with these modifications. This nav needs more work in general but this at least fixes the immediate problem of making the utility links available. Thank you! 

Also looping in @serenamarie125

**Comment by andresgalante** [comment]
Forgot to tag @rhamilto to this PR.

**Comment by serenamarie125** [comment]
@andresgalante I find it a bit odd that the visualization for primary nav items and the utilities is the same, but agree this is a step in the right direction.  In your example, you have notification drawer in the masthead, but it doesn't go to the vertical navigation ... is it possible to have any masthead utility be included on the vertical nav?

**Comment by rhamilto** [comment]
Seems practical to me.  👍

**Comment by andresgalante** [comment]
@serenamarie125 notification drawer is out of scope for [this story](https://patternfly.atlassian.net/browse/PTNFLY-1934). I think any utility link can be added the way this PR structures it. The drawer could work if it jumps to a new page with the notifications.

**Comment by serenamarie125** [comment]
@andresgalante Yes understood that notification drawer is out of scope for this PR.  I was just curious if it's possible for someone to customize the code and add other items, if they desired to.

**Review by serenamarie125 — APPROVED** [review]
Look good, thanks @andresgalante

**Review by jeff-phillips-18 — APPROVED** [review]
👍

**Comment by LHinson** [comment]
Thanks @rhamilto @serenamarie125 @jeff-phillips-18 @andresgalante . Merging now.

**Pull request merged**